File: my-code/DSA/JS/Add2.js
function addUpTo(n) {
  return (n * (n + 1)) / 2;
}

let startTime = performance.now();
console.log(addUpTo(1000000000));
let endTime = performance.now();
console.log(`The running time is ${(endTime - startTime) / 1000} seconds. `);

File: my-code/DSA/PYTHON/Add2.py
import time

def add_up_to(n):
    return n * (n + 1) / 2

start_time = time.perf_counter()
print(add_up_to(1000000000))
end_time = time.perf_counter()
print(f'The running time is {(end_time - start_time)} seconds. ')